web-animations-1/Overview.bs

```diff
@@ -957,7 +990,9 @@ an animation, <var>animation</var>, to <var>seek time</var> is as follows:
             <a lt="inactive timeline">inactive</a>, or
         *   <var>animation</var>'s [=playback rate=] is 0,
 
-    ::  Set <var>animation</var>'s <a>hold time</a> to <var>seek time</var>.
+    ::  Set <var>animation</var>'s <a>hold time</a> to <var>seek time</var>
+    ::  Set <var>animation</var>'s <a>hold phase</a> to
+        [=timeline active phase|active=].
```

Comment:
This should probably be a numbered list. Let's add the missing '.' from the first step and update to more modern bikeshed syntax at the same time:

```bikeshed
:: 1. Set |animation|'s [=hold time=] to |seek time|.
   1. Set |animation|'s [=hold phase=] to [=timeline active phase|active=].
```